ops/scripts/create_dev_sandbox.sh

```diff
@@ -112,10 +112,14 @@ sed -i '' '/          - development/ {a\
           - '"$1"'
 }' .github/workflows/reset-db.yaml
 
-sed -i '' '/          - development/ {a\
+sed -i '' '/          - backup/ {a\
```

Comment:
**blocking change**  why did you change this? this tells it where to add stuff below, and we don't want to add it below developement, we want things added below development. please revert this.

Reply:
Sorry - meant to add new sandboxes under backup in the options in deploy-manual and not deploy. That was an oversight on my part so thank you for catching that!